app/assets/javascripts/fae/navigation/_subnav_highlighter.js

```diff
@@ -69,21 +69,10 @@ Fae.navigation.subnav_highlighter = {
 
       $('.js-highligher').last().addClass('-active').removeClass('js-highligher');
     }
-    FCH.scroll.push(scrollCallback);
 
-    //highlight the first one on page load
+    // highlight the first one on page load
     scrollCallback();
-
-    /**
-     * On resize, ensure last section is the same as the window height so when we skip to it the label is at the top
-     * @private
-     */
-    function lastSectionBuffer() {
```

Comment:
Heads up, this is a request the PDs *love* to make without considering the other edge cases. Removing this will mean that the fancy sub nav indicator (the colored 3px border that slides over when you scroll through sections on long forms) will **not** trigger on the very last section. This function was written to ensure that when the user clicks on the last label in the subnav, the scroll will take them to that the actual start of that section. It's required to match the behavior of the other links in the subnav.